addons/europeans_tribe.wad/productionsites/clay_pit/init.lua

```diff
@@ -82,8 +82,8 @@ descriptions:new_productionsite_type {
          }
       },
       fish_breeding_in_sea = {
-         -- TRANSLATORS: Completed/Skipped/Did not start breeding fish because ...
-         descname = _"breeding fish",
+         -- TRANSLATORS: Completed/Skipped/Did not start breeding fish in sea because ...
+         descname = _"breeding fish in sea",
```

Comment:
Suggestion:
```suggestion
         -- TRANSLATORS: Completed/Skipped/Did not start breeding fish in the sea because ...
         descname = _"breeding fish in the sea",
```

Reply:
Solved